README.rst

```diff
@@ -87,6 +87,24 @@ If something ever goes wrong - a request times out, the ``sixpack`` server disap
 calls will return the control alternative, and all ``convert`` calls will seem successful (and we'll note this happend
 in the log).
 
+-----------------
+Tracking Locally
+-----------------
+
+By passing in the argument `local` in the SixpackTest constructor you can tell django-sixpack to create a convertible db record for each participant.
```

Comment:
Can you add docs about why you'd want to do this/use cases for this? I get that it's creating a record in the DB, but not why I'd like to do that.